### ISSUE TITLE
Performace of agent degrades as training continues [APEX]

### ISSUE THREAD

**Issue by mynkpl1998** [ray-project/ray#4186]
### System information
- **OS Platform and Distribution**: Linux Ubuntu 16.04
- **Ray installed from (source or binary)**: Binary
- **Ray version**: 0.6.2
- **Python version**: 3.6

### Describe the problem

Hi, I am using APEX agent to solve my problem. In the initial part of the training, policy shows good performance. However, it degrades as training continues. I think this is because of the fact that I am not setting hyper-parameters correctly. Can anyone point me to the hyper-parameter(s) to play around with to get the correct behaviour? Thanks

### Source code / logs
**Attaching Training Statistics to demonstrate the degradation of the performance.**
The graphs given below are of a two runs. However, I tried to run many experiments; all of them showed similar behaviour.

**Run 1**
![image](https://user-images.githubusercontent.com/15110441/53512605-2be91f80-3ae9-11e9-8db3-c0f18dd58ee0.png)

**Run 2**
![image](https://user-images.githubusercontent.com/15110441/53512380-9e0d3480-3ae8-11e9-9fc4-cde4ffe229ff.png)

**Training Configuration File**
```
IDM:
  env: tsim-v0
  run: APEX
  checkpoint_freq: 50
  config:
    n_step: 30
    num_gpus: 1
    num_workers: 9
    sample_batch_size: 50
    horizon: 1500
    train_batch_size: 512
    target_network_update_freq: 50000
    callbacks:
      on_episode_end: None
```

**Comment by ericl** [comment]
Hi, it's hard to provide a definitive answer here, but I would try making sure you have adequate exploration at the end of the schedule, or annealing down your learning rate.

**Comment by mynkpl1998** [comment]
Thanks, @ericl for the suggestion. However, the issue was not with rllib. There was a bug in environment code. Apologies. Closing the issue.
